src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java

```diff
@@ -64,6 +65,10 @@ public String getDefaultDriver(final String url) {
         return null;
     }
 
+    public MongoDatabase getMongoDatabase() {
```

Comment:
This looks like a helpful addition to this class - e.g. can be used when writing a `CustomTaskChange` after casting the `Database` to `MongoLiquibaseDatabase`